stories/chats/src/BubblePlainText.tsx

```diff
@@ -1,18 +1,10 @@
 import * as React from 'react';
-import {
-    TouchableWithoutFeedback,
-    StyleSheet,
-    Platform,
-    View,
-    Animated,
-    TextStyle,
-    I18nManager,
-} from 'react-native';
+import { StyleSheet, Platform, View, Animated, TextStyle, I18nManager } from 'react-native';
```

Comment:
Animated from react-native is still used here (and as far as I see this component is transforming which is more likely to lead to the bug)

Reply:
Completely removed